docs/synctl-update-test.md

```diff
@@ -33,25 +34,35 @@ synctl update test <id> [options]
     --operation <method>               HTTP request methods, GET, POST, HEAD, PUT, etc.
     --mark-synthetic-call <boolean>    set markSyntheticCall
     --url <url>                        HTTP URL
+    --headers <json>                   HTTP headers
+    --body <string>                    HTTP body
     --follow-redirect <boolean>        set follow-redirect
     --expect-status <int>              set expected HTTP status code
     --validation-string <string>       set validation-string
+    --expect-status <int>              expected status code, Synthetic test will fail if response status is not equal to it, default 200
+    --expect-json <string>             An optional object to be used to check against the test response object
+    --expect-match <string>            An optional regular expression string to be used to check the test response
+    --expect-exists <string>           An optional list of property labels used to check if they are present in the test response object
+    --expect-not-empty <string>        An optional list of property labels used to check if they are present in the test response object with a non-empty value
+    --allow-insecure <boolean>         if set to true then allow insecure certificates
 ```
 
 ### Options for API Script test
 ```
-    --file,-f <file-name>              json payload
-    --script-file <file-name>          specify a script file to update APIScript or BrowserScript
+    --file <file-name>                 json payload
```

Comment:
`--file` this option has been removed. so just remove this option from every example. just use `-f` or `--from-file` as an alternative.